src/main/java/com/zendesk/maxwell/schema/MysqlSavedSchema.java

```diff
@@ -635,27 +634,29 @@ public Position getPosition() {
 	private void fixUnsignedColumns(Schema recaptured) throws SQLException, InvalidSchemaError {
```

Comment:
I'm thinking we could just get rid of some of this `fixXXX` code.  Any installations that had to have data fixed this way are either fixed already or long dead.

Reply:
Some orgs are way behind and rarely upgrade making long-term upgrade path maintenance a really great feature. I'd rather keep it if that's okay. I've definitely had the misfortune of working places with 8+ year old external dependencies. Continuous upgrade paths make life much easier for the few people willing to improve things.